Assets/Script/PositioningTheUI.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEngine.XR;
+
+public class PositioningTheUI : MonoBehaviour
+{
+    public XRNode xrNode = XRNode.Head; // XRNode for user position and rotation
+    public float distanceFromUser = 2f; // Distance of the UI from the user
+    public float heightFromGround = 1.5f; // Height above the ground where the UI should appear
+
+
+    void Start()
+    {
+        // Check if XRNode is valid
+        if (XRSettings.isDeviceActive)
+        {
+            // Get the position and rotation of the XR Node (e.g., headset)
+            InputTracking.GetNodeStates(new List<XRNodeState>());
```

Comment:
InputTracking ist veraltet -> siehe Projektwarnung/Warnung in Visual Studio Code, Ersatz wird dort vorgeschlagen